projects/plugins/jetpack/_inc/lib/class-jetpack-podcast-helper.php

```diff
@@ -98,8 +98,8 @@ function ( $track ) {
 	/**
 	 * Gets a specific track from the supplied feed URL.
 	 *
-	 * @param string $guid     The GUID of the track.
-	 * @return array|WP_Error  The track object or an error object.
+	 * @param string $guid    The GUID of the track.
+	 * @return {array|WP_Error}  The track object or an error object.
```

Comment:
I might just revert the changes from this file to keep the changeset cleaner (non blocking).